src/scripts/browserDeviceProfile.js

```diff
@@ -930,7 +938,7 @@ export default function (options) {
         av1VideoRangeTypes += '|HLG';
     }
 
-    if (supportsDolbyVision(options)) {
+    if (supportsDolbyVision(options) && canPlayDolbyVisionHevc(videoTestElement)) {
```

Comment:
Iirc, `canPlayType` doesn't work on all platforms. So we probably shouldn't call `canPlayDolbyVisionHevc` on Tizen and webOS.

Reply:
IIRC Samsung only seems to be focused on promoting their HDR10 Plus, if they enable DolbyVision in the future we could add it back.

Reply:
I'm wondering how the TV client utilizes this boolean `options.supportsDolbyVision`? Is there a button I didn't notice?

Reply:
It is filled by native client (app only).
https://github.com/jellyfin/jellyfin-webos/blob/3cf2c835fed6017bbe61ea81ad962ed1827c0d0d/frontend/js/webOS.js#L80
https://github.com/jellyfin/jellyfin-webos/blob/3cf2c835fed6017bbe61ea81ad962ed1827c0d0d/frontend/js/index.js#L20

Reply:
Hey, just chiming in to confirm this works fine on my 2022 webos.